ios/RN/RNCamera.m

```diff
@@ -17,7 +17,7 @@ @interface RNCamera ()
 
 @property (nonatomic, strong) RCTPromiseResolveBlock videoRecordedResolve;
 @property (nonatomic, strong) RCTPromiseRejectBlock videoRecordedReject;
-@property (nonatomic, strong) RNFaceDetectorManager *faceDetectorManager;
+@property (nonatomic, strong) id faceDetectorManager;
```

Comment:
why `id`?

Reply:
id is a generic pointer in objc, meaning it can point to any ObjC class.

Since faceDetectormanager can be of FaceDetectorManager or FaceDetectorManagerStub, it should be id.

Reply:
great to know, tks